### PR TITLE
fix: 403 处理

### PR DESCRIPTION
**Pull request by leafage-collb**
- 403 处理

### CODE REVIEW

**Review by huangpeng0428 — APPROVED** [review]
LGTM

File: src/dashboard-front/src/language/lang/en.js [hunk 1]
```diff
@@ -1208,5 +1208,8 @@ export default {
     "请输入关键字搜索": "Please input keyword",
     "API Gateway | 腾讯蓝鲸智云": "API Gateway | BlueKing",
     "只支持.tar.gz、.tgz、.zip 压缩格式": "Only supports .tar.gz, .tgz, .zip compression formats",
-    "未知错误": "Unknown mistake"
```
Comment (alex-smile) [inline src/dashboard-front/src/language/lang/en.js:1211]: Unknown error

File: src/dashboard-front/src/language/lang/en.js [hunk 2]
```diff
@@ -1208,5 +1208,8 @@ export default {
...
+    "未知错误": "Unknown mistake",
+    "无该应用访问权限": "No access permission for this application.",
```
Comment (alex-smile) [inline src/dashboard-front/src/language/lang/en.js:1212]: No access permission to this application.

File: src/dashboard-front/src/language/lang/en.js [hunk 3]
```diff
@@ -1208,5 +1208,8 @@ export default {
...
+    "服务异常": "Service exception",
+    "无法连接到后端服务，请稍候再试。": "Unable to connect to backend service, please try again later."
```
Comment (alex-smile) [inline src/dashboard-front/src/language/lang/en.js:1214]: Unable to access backend service, please try again later.

File: src/dashboard-front/src/main.js [hunk 4]
```diff
@@ -92,6 +92,28 @@ auth.requestCurrentUser().then(async (user) => {
   } else {
     auth.redirectToLogin()
   }
-}, _ => {
-  auth.redirectToLogin()
+}, err => {
+  let title = '服务异常'
+  let status = err.response.status || 500
+  let message = '无法连接到后端服务，请稍候再试。'
```
Comment (alex-smile) [inline src/dashboard-front/src/main.js:98]: 访问后端服务出现错误，请稍后再试。

Access backend service error, please try again later.

**Review by alex-smile — APPROVED** [review]

File: src/dashboard-front/src/main.js [hunk 5]
```diff
@@ -92,6 +92,28 @@ auth.requestCurrentUser().then(async (user) => {
...
+  const { data } = err.response
+  if (err.code === 'PRODUCT_NOT_READY') {
+    message = err.detail || err.message
+  } else if (status === 403 && data.code === 1302403) {
+    status = 403
+    title = '无该应用访问权限'
+    message = data.detail || err.detail || err.message
+  } else {
+    // 重新登录
+    auth.redirectToLogin()
```
[the remark this answers is not in the document]
Comment (leafage-collb) [reply src/dashboard-front/src/main.js:110]: 后面的commit已经处理

**Pull request merged**